frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -130,6 +129,7 @@ export const infiniteListLogic = kea<infiniteListLogicType>([
                         offset,
                         excluded_properties: JSON.stringify(excludedProperties),
                         properties: propertyAllowList ? propertyAllowList.join(',') : undefined,
+                        ...(props.hideBehavioralCohorts ? { hide_behavioral_cohorts: 'true' } : {}),
```

Comment:
add the query param if the prop tells you to (most props don't this prop is just used in the feature flag property filter dropdown).